### ISSUE TITLE
[Bug]: "path" broken in 0.17.1 for NX-OS 9.3(9)

### ISSUE THREAD

**Issue by supertylerc** [netenglabs/suzieq#702]
### Suzieq version

0.17.1

### Install Type

container

### Python version

whatever is bundled with your container

### Impacted component

multiple components

### Steps to Reproduce

0. Be on suzieq 0.17.1/latest tag on Docker container
1. Deploy a basic VXLAN EVPN fabric on NX-OS 9.3(9)
2. Run the poller
3. Assert OSPF, BGP, evpnVni, arpnd
4. Try to use `path show src='1.2.3.4' dest='4.3.2.1' namespace='lab'`

### Expected Behavior

Path will be returned

### Observed Behavior

Receive the following error:

```0  ERROR: invalid literal for int() with base 10: ''```

### Additional Context

The exact same devices, NOS versions, topology, and configuration works with the Docker image tag `0.17.0`.

**Comment by ddutt** [comment]
Please try with 0.17.1. If the issue occurs even with that version, please do the following and report back the output:
```
set debug=True
path show src='1.2.3.4' dest='4.3.2.1' namespace=lab
```

Thanks

**Comment by supertylerc** [comment]
Ah, perfect!  Was trying to figure out how to get more debug output and increasing verbosity wasn't doing it for me.  Will tuck that away for later, thank you!  Here's the traceback:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/suzieq/cli/sqcmds/command.py", line 501, in _invoke_sqobj
    df = fn(**kwargs)
  File "/usr/local/lib/python3.7/site-packages/suzieq/sqobjects/basicobj.py", line 206, in get
    return self.engine.get(**kwargs)
  File "/usr/local/lib/python3.7/site-packages/suzieq/engines/pandas/path.py", line 760, in get
    self._init_dfs(self.namespace, src, dest)
  File "/usr/local/lib/python3.7/site-packages/suzieq/engines/pandas/path.py", line 45, in _init_dfs
    'vlan', 'master', 'macaddr', 'timestamp']) \
  File "/usr/local/lib/python3.7/site-packages/suzieq/sqobjects/basicobj.py", line 206, in get
    return self.engine.get(**kwargs)
  File "/usr/local/lib/python3.7/site-packages/suzieq/engines/pandas/interfaces.py", line 65, in get
    df = self._add_portmode(df, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/suzieq/engines/pandas/interfaces.py", line 596, in _add_portmode
    df['vlan'] = df.vlan.astype(int)
  File "/usr/local/lib/python3.7/site-packages/pandas/core/generic.py", line 5815, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors)
  File "/usr/local/lib/python3.7/site-packages/pandas/core/internals/managers.py", line 418, in astype
    return self.apply("astype", dtype=dtype, copy=copy, errors=errors)
  File "/usr/local/lib/python3.7/site-packages/pandas/core/internals/managers.py", line 327, in apply
    applied = getattr(b, f)(**kwargs)
  File "/usr/local/lib/python3.7/site-packages/pandas/core/internals/blocks.py", line 591, in astype
    new_values = astype_array_safe(values, dtype, copy=copy, errors=errors)
  File "/usr/local/lib/python3.7/site-packages/pandas/core/dtypes/cast.py", line 1309, in astype_array_safe
    new_values = astype_array(values, dtype, copy=copy)
  File "/usr/local/lib/python3.7/site-packages/pandas/core/dtypes/cast.py", line 1257, in astype_array
    values = astype_nansafe(values, dtype, copy=copy)
  File "/usr/local/lib/python3.7/site-packages/pandas/core/dtypes/cast.py", line 1174, in astype_nansafe
    return lib.astype_intsafe(arr, dtype)
  File "pandas/_libs/lib.pyx", line 679, in pandas._libs.lib.astype_intsafe
ValueError: invalid literal for int() with base 10: ''

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/nubia/internal/cmdbase.py", line 408, in run_interactive
    ret = fn(**args_dict)
  File "/usr/local/lib/python3.7/site-packages/suzieq/cli/sqcmds/command.py", line 179, in show
    **self.lvars,
  File "/usr/local/lib/python3.7/site-packages/suzieq/cli/sqcmds/command.py", line 507, in _invoke_sqobj
    raise Exception from ex
Exception
```

**Comment by ddutt** [comment]
Interesting. Would you be able to provide me with either the parquet output or a test data, I can verify the error. The fix is simple, but I dont want to fix the symptom without finding the cause first.

**Comment by supertylerc** [comment]
Sorry for the delay!  Will try to get you that data today/tomorrow.

**Comment by ddutt** [comment]
Please test the 0.17.2rc1 release and see if it fixes the issue.

**Comment by ddutt** [comment]
Should be fixed with commit bee05b25bc492d6ecad3a6a91518440e29ef1cb2
